std/src/main/scala/algebra/std/map.scala

```diff
@@ -130,3 +133,25 @@ class MapRng[K, V](implicit V: Rng[V]) extends MapSemiring[K, V] with Rng[Map[K,
   override def minus(x: Map[K, V], y: Map[K, V]): Map[K, V] =
     subtractMap(x, y)(V.minus)(V.negate)
 }
+
+class MapGenBool[K, V](implicit GBV: GenBool[V], EqV: Eq[V]) extends GenBool[Map[K, V]] {
```

Comment:
I guess this passes because `Eq[Map[K, V]]` is the `MapVectorEq`, and since the guess there is an `AdditiveGroup[Set[Byte]]` somewhere? It's a bit unclear to me.

I wonder if it makes sense that this class also extend or have a `Eq[Map[K, V]]` which makes it clear that `GBV.zero` is the some as a missing value. I get really worried that we have action at a distance here: laws that require that the Eq instance in sync.